src/schema/v2/homeView/sectionTypes/NavigationPills.ts

```diff
@@ -35,6 +36,10 @@ const NavigationPillType = new GraphQLObjectType<
       type: new GraphQLNonNull(GraphQLString),
       description: "The context module for analytics",
     },
+    icon: {
+      type: GraphQLString,
+      description: "Icon file name",
+    },
```

Comment:
I debated typing this as an enum. But since the source of truth for which icons is supported is over in Eigen, there didn't seem to be any benefit it doing so, just an extra chore to keep the two repos synchronized.